packages/blocks/src/__tests__/schema/__snapshots__/schema.unit.spec.ts.snap

```diff
@@ -0,0 +1,794 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`serialized block schema > matches current snapshot 1`] = `
+[
+  "{
+  "flavour": "affine:code",
+  "metadata": {
+    "version": 1,
+    "role": "content",
+    "parent": [
+      "affine:note",
+      "affine:paragraph",
+      "affine:list",
+      "affine:edgeless-text"
+    ],
+    "children": []
+  },
+  "props": {
+    "text": {
+      "type": "text"
+    },
+    "language": {
+      "type": "string",
+      "default": "Plain Text"
+    },
+    "caption": {
+      "type": "string",
+      "default": ""
+    }
+  }
+}",
```

Comment:
The indent seems not correct

Reply:
It's auto generated by vitest, I don't think the prettier config applies here.